README.md

```diff
@@ -7,13 +7,16 @@
 
```

Comment:
- [ ] Kindy change your description to reflect the status of this milestone, it's supposed to be an individual project, and going by your commits you did it alone. Also, do the same to your pull request summary.